### PR TITLE
chore: cleanup old textfield component

### PR DESCRIPTION
**Pull request by Onibenjo**

### CODE REVIEW

File: src/components/TextField/MuiTextField.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Color } from '@mui/material';
+import { Color, Theme } from '@mui/material';
```
Comment (cingopablo) [inline src/components/TextField/MuiTextField.tsx:1]: Why don't we put all this inside the `TextField.tsx` and just export it as:
`export const TextField = () => whatever`

This way we would delete one file and won't be needing to do this:
`export { MuiTextField as TextField } from './MuiTextField';`

Comment (cingopablo) [reply src/components/TextField/MuiTextField.tsx:1]: Probably there's something I'm missing about how we use this component 🤓

Comment (Onibenjo) [reply src/components/TextField/MuiTextField.tsx]: nah, you aren't missing anything...
I was already updating that to deleting the files

Comment (Onibenjo) [reply src/components/TextField/MuiTextField.tsx]: We did an incremental update
So `TextField` and `MuiTextField` are required tho

Comment (cingopablo) [reply src/components/TextField/MuiTextField.tsx]: Ah got it, thanks!

**Review by cingopablo — APPROVED** [review]

**Pull request merged**